### ISSUE TITLE
Model inference with SVD

### ISSUE THREAD

**Issue by zorian15** [matsengrp/torchdms#83]
Train torchdms models with a low-rank approximation of beta latent spaces after each gradient descent step.
